pkg/volume/gce_pd/gce_util.go

```diff
@@ -109,7 +111,11 @@ func (gceutil *GCEDiskUtil) CreateVolume(c *gcePersistentDiskProvisioner) (strin
 			zonesPresent = true
 			configuredZones = v
 		case "replication-type":
-			replicationType = strings.ToLower(v)
+			if utilfeature.DefaultFeatureGate.Enabled(features.GCERegionalPersistentDisk) {
+				replicationType = strings.ToLower(v)
+			} else {
+				return "", 0, nil, "", fmt.Errorf("invalid option %q for volume plugin %s", k, c.plugin.GetPluginName())
```

Comment:
Should indicate in error that the feature gate is disabled, and should be enabled if you want to use this feature.